scm-ui/ui-api/src/admin.ts

```diff
@@ -22,14 +22,17 @@
  * SOFTWARE.
  */
 
-import { ApiResult, useRequiredIndexLink } from "./base";
+import { ApiResult, useIndexLink } from "./base";
 import { UpdateInfo } from "@scm-manager/ui-types";
 import { useQuery } from "react-query";
 import { apiClient } from "@scm-manager/ui-components";
 
 export const useUpdateInfo = (): ApiResult<UpdateInfo | null> => {
-  const indexLink = useRequiredIndexLink("updateInfo");
-  return useQuery<UpdateInfo | null, Error>("updateInfo", () =>
-    apiClient.get(indexLink).then(response => (response.status === 204 ? null : response.json()))
+  const indexLink = useIndexLink("updateInfo");
+  return useQuery<UpdateInfo | null, Error>(
+    "updateInfo",
+    () =>
+      apiClient.get(indexLink ? indexLink : "").then((response) => (response.status === 204 ? null : response.json())),
```

Comment:
An `apiClient.get("")` would fetch the index. I don't think that this is the correct behaviour, throwing an error should be much better in this case.

Reply:
You are absolutely right! I thought in this case it wouldn't be a big problem because the query is disabled anyway.